### PR TITLE
Add 'Managing the audit daemon' page

### PR DESCRIPTION
**Pull request by travier**
See: https://github.com/coreos/fedora-coreos-tracker/issues/1362
See: https://github.com/linux-audit/audit-userspace/commit/39802bffbfc62501461c916d9ccf748afdff7d94

### CODE REVIEW

File: modules/ROOT/pages/audit.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+= Managing the audit daemon (`auditd`)
+
+Starting with the first release based on Fedora 39, Fedora CoreOS includes the audit daemon (`auditd`) to load and manage audit rules.
+
+While the audit daemon
```
Comment (dustymabe) [inline modules/ROOT/pages/audit.adoc:5]:
```suggestion
```

File: modules/ROOT/pages/audit.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Like all system daemons on Fedora CoreOS, the audit daemon is managed by systemd but with an exception: it can not be stopped or restarted via `systemctl stop auditd` or `systemctl restart auditd` for compliance reasons.
+
+From [Unable to restart/stop auditd service using systemctl command in RHEL](https://access.redhat.com/solutions/2664811):
```
Comment (dustymabe) [inline modules/ROOT/pages/audit.adoc:8]: Fix the link.. I think this is markdown formatting, not asciidoc.

File: modules/ROOT/pages/audit.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+> The reason for this unusual handling of restart/stop requests is that auditd is treated specially by the kernel: the credentials of a process that sends a killing signal to auditd are saved to the audit log. The audit developers do not want to see the credentials of PID 1 logged there. They want to see the login UID of the user who initiated the action.
+
+To stop and restart the audit daemon, you should use the following commands:
+
+[source,bash]
+----
+$ sudo auditctl stop
+$ sudo systemctl start  # Only if you want to restart the daemon
```
Comment (dustymabe) [inline modules/ROOT/pages/audit.adoc:17]: Maybe more clear:

```suggestion
$ sudo systemctl start  # Only if you want it started again
```

File: modules/ROOT/pages/audit.adoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+----
+
+You man also use the following commands to reload the rules, rotate the logs, resume (?), fetch the state:
```
Comment (dustymabe) [inline modules/ROOT/pages/audit.adoc:20]:
```suggestion
You may also use the following commands to reload the rules, rotate the logs, resume (?), fetch the state:
```

Maybe we could link to docs somewhere for exactly what each of those commands do?

**Comment by travier** [comment]
Thanks. Updated.

File: modules/ROOT/pages/audit.adoc [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+----
+
+You may also use the following commands to reload the rules, rotate the logs, resume logging or dump the daemon state:
+
+[source,bash]
+----
+$ sudo auditctl reload
+$ sudo auditctl rotate
+$ sudo auditctl resume
+$ sudo auditctl state
```
[the remark this answers is not in the document]
Comment (travier) [reply modules/ROOT/pages/audit.adoc]: Thanks!

File: modules/ROOT/nav.adoc [hunk 6]
```diff
@@ -42,6 +42,7 @@
 ** xref:counting.adoc[Node counting]
 ** xref:time-zone.adoc[Configuring Time Zone]
 ** xref:grub-password.adoc[Setting a GRUB password]
+** xref:audit.adoc[Managing the audit daemon]
```
[the remark this answers is not in the document]
Comment (travier) [reply modules/ROOT/nav.adoc:45]: Agree, though the same. I still haven't updated my "re-org" PR that could help with this.

**Review by jlebon — APPROVED** [review]

**Review by dustymabe — APPROVED** [review]
LGTM

**Comment by travier** [comment]
Merging now that beta is out.

**Pull request merged**